pyproject.toml

```diff
@@ -22,14 +22,14 @@ exclude = '''
   | datadog_checks/dev/tooling/templates
   # Vendored third party libraries
   | datadog_checks/[^/]+/vendor
-)/
+    )/
   |
 # Files
-(
+    (
     # TODO: remove when upstream addresses https://github.com/DataDog/integrations-core/blob/c71e6d7204192a8002109da92452003598df2d28/datadog_checks_dev/datadog_checks/dev/tooling/signing.py#L9-L14
     datadog_checks/dev/tooling/signing\.py$
-)
-'''
+    )
+    '''
```

Comment:
Mypy seems to have some issues parsing `toml` files. I had to update that part. Same issue mentioned [here](https://github.com/pydantic/pydantic/issues/2895#issuecomment-990782424)